wrapper/src/test/java/integration/util/AuroraTestUtility.java

```diff
@@ -276,16 +276,21 @@ public String createAuroraCluster() throws InterruptedException {
     // Create Instances
     for (int i = 1; i <= numOfInstances; i++) {
       final String instanceName = dbIdentifier + "-" + i;
-      rdsClient.createDBInstance(
-          CreateDbInstanceRequest.builder()
-              .dbClusterIdentifier(dbIdentifier)
-              .dbInstanceIdentifier(instanceName)
-              .dbInstanceClass(dbInstanceClass)
-              .engine(dbEngine)
-              .engineVersion(dbEngineVersion)
-              .publiclyAccessible(true)
-              .tags(testRunnerTag)
-              .build());
+      CreateDbInstanceRequest.Builder requestBuilder = CreateDbInstanceRequest.builder()
+          .dbClusterIdentifier(dbIdentifier)
+          .dbInstanceIdentifier(instanceName)
+          .engine(dbEngine)
+          .engineVersion(dbEngineVersion)
+          .publiclyAccessible(true)
+          .tags(testRunnerTag);
+
+      // The latest PG version is limitless, which is not compatible with dbInstanceClass (db.r5.large), so we will
+      // leave the instance class unspecified in this scenario.
+      if (!dbEngineVersion.contains("limitless")) {
```

Comment:
Can we create an internal boolean flag isEngineLimitless and use it here? We can set this flag in createCluster()